src/main/java/dev/carrico/repos/StackLinkRepo.java

```diff
@@ -0,0 +1,15 @@
+package dev.carrico.repos;
+
+import dev.carrico.entities.StackLink;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+
+import java.util.Set;
+
+@Component
+@Repository
+public interface StackLinkRepo extends CrudRepository<StackLink, Integer> {
+    Set<StackLink> findByLearnerId(int learnerId);
+    Set<StackLink> findByRelationship(String relationship);
```

Comment:
This should be learnerId and relationship maybe?

Reply:
👍 Was gonna ask you guys what you thought. We might not need the relationship one at all